shipyard-controller/main.go

```diff
@@ -293,6 +301,8 @@ func _main(env config.EnvConfig, kubeAPI kubernetes.Interface) {
 		}
 	}()
 
+	debugEngine.Run("localhost:9090")
```

Comment:
the call to the `Run()` method is blocking, so everything else after this will not be executed - please move this into a new goroutine